glsk/glsk-document-api/src/main/java/com/powsybl/glsk/api/util/converters/GlskPointScalableConverter.java

```diff
@@ -68,9 +68,6 @@ public static Scalable convert(Network network, List<AbstractGlskShiftKey> shift
                 throw new GlskException("In convert glskShiftKey business type not supported");
             }
         }
-        if (percentages.isEmpty()) {
-            return new EmptyScalable();
-        }
```

Comment:
Shouldn't we keep this EmptyScalable feature ?

Reply:
Since `ProportionalScalable` are now allowed to be empty, I think they have the wanted behavior?

Reply:
Indeed I forget this part ! :)